ios/Classes/DocumentViewFactory.m

```diff
@@ -0,0 +1,28 @@
+#import "FlutterDocumentView.h"
+#import "PdftronFlutterPlugin.h"
+#import "PTPluginUtils.h"
+
+@implementation DocumentViewFactory {
+    NSObject<FlutterBinaryMessenger>* _messenger;
+}
+
+- (instancetype)initWithMessenger:(NSObject<FlutterBinaryMessenger> *)messenger
+{
+    self = [super init];
+    if (self) {
+        _messenger = messenger;
+    }
+    return self;
+}
+
+- (NSObject<FlutterMessageCodec> *)createArgsCodec
+{
+    return [FlutterStandardMessageCodec sharedInstance];
+}
+
+- (NSObject<FlutterPlatformView> *)createWithFrame:(CGRect)frame viewIdentifier:(int64_t)viewId arguments:(id)args
```

Comment:
args is completely ignored here?

Reply:
It was not used before in DocumentView as well, so yeah